server/server.js

```diff
@@ -129,6 +129,7 @@ function setUpRoutes() {
     var locale = req.cookies.lang  || req.acceptsLanguages(['fi', 'sv', 'en']) || 'en';
     var messages = translations[locale]
     var context = application.createContext()
+    navigator = {userAgent:req.headers['user-agent']};  //required by material-ui
```

Comment:
Server side rendering of material-ui components needs the global 'navigator' to be set